src/changes/changes.xml

```diff
@@ -66,6 +66,7 @@ The <action> type attribute can be add,update,fix,remove.
     <action                   type="fix" dev="ggregory" due-to="CiprianBodnarescu">Use Set instead of List for checking the contains() method #734.</action>
     <action                   type="fix" dev="kinow" due-to="Roland Kreuzer">Javadoc for StringUtils.substringBefore(String str, int separator) doesn't mention that the separator is an int.</action>
     <!-- ADD -->
+    <action issue="LANG-1677" type="add" dev="debae" due-to="Dennis Baerten">Add ReflectionDiffBuilder.setExcludeFieldNames(...) and DiffExclude annotation to support field exclusion from diff.</action>
```

Comment:
Don't edit this file, it will usually clash depending on parallel changes.

Reply:
Removed the file again.